src/main/java/dev/jbang/source/Project.java

```diff
@@ -163,7 +163,7 @@ public Project setDescription(String description) {
 		return this;
 	}
 
-	@Nullable
+	@Nonnull
```

Comment:
how can it be non null?

Reply:
Because `Optional.ofNullable(xxx)` will always return a value, never `null`

Reply:
Ah ofcourse. Its nonnull optional. Brainfart.